config/checkstyle-input-checks.xml

```diff
@@ -24,12 +24,12 @@
 
   <module name="TreeWalker">
     <module name="Regexp">
-      <property name="id" value="StyleValidationCommentInInputs"/>
+      <property name="id" value="UnnecessaryOkComment"/>
       <property name="format"
-               value="(\/\/ (ok|violation|\d violations|filtered violation))"/>
-      <message key="required.regexp"
-             value="All check input files in the ''src/(it|test)/resources*/'' folder must
-                   have trailing comments &quot;// ok&quot; and/or &quot;// violation&quot;."/>
+               value="(?i)\/\/ ok$"/>
```

Comment:
What is `(?i)`?

Reply:
Case insensitive mode flag.

Reply:
We should make a new issue for this.

`RegexpMultiline` and `RegexpSingleline` and `RegexpSinglelineJava` has `ignoreCase` but for some reason `RegExp` does not.

Reply:
I think we added such properties as we did not know it was possible to be done by regexp. It was big surprise for me that such mode exists.

Do we really need to add  such property?

Reply:
I do not think we such a property, but we should probably have some examples with this flag to demonstrate how to activate case insensitive mode.